### PR TITLE
Add SUPPORT.md & link from README

### PR DESCRIPTION
**Pull request by hflynn**
See https://trello.com/c/Gt8CIWJ0/598-support-files

### CODE REVIEW

**Comment by hflynn** [comment]
Link is working now so Travis failure is unrelated to this PR

**Comment by sbesson** [comment]
Travis errors seems to be E722 and E741. But I see a lot of open PRs are already suffering from these. For the upcoming 3.2.0 release, we'll want to get this fixed prior to integrating the ongoing work. @will-moore @jburel is there already an open PR or staging branch containing the necessary fixes?

**Comment by will-moore** [comment]
https://github.com/ome/omero-figure/pull/251 contains flake8 fixes

**Comment by sbesson** [comment]
Thanks. #251 is currently conflicting and needs some review as far as I can tell so I would not tie the changes to this feature PR. Either the flake fixes need to be open as a standalone PR or we could to cherry-pick the changes here and fix `master`. 
Main concern about https://github.com/ome/omero-figure/commit/14b780756fc986c5643ab0bf23d185c40f29f5d0 is that I am unclear about the dropping of the `try/except` blocks and the dropped `.getValue()` - do these require testing?

**Comment by jburel** [comment]
Being more explicit in the ``try/except`` should have be done from the start in my opinion
I would not have expected the ``getValue`` to be removed

File: README.rst [hunk 1]
```diff
@@ -12,6 +12,7 @@ An OMERO.web app for creating figures from images in OMERO.
 
 For full details see http://figure.openmicroscopy.org
 
+Also see [SUPPORT.md](./SUPPORT.md)
```
Comment (jburel) [inline README.rst]: I will simply it and only indicate 
```
For full details see [SUPPORT.md](./SUPPORT.md)
```

**Comment by hflynn** [comment]
@jburel edit pushed

**Comment by jburel** [comment]
Thanks @hflynn  merging

**Pull request merged**